src/test/java/com/awesomeorg/airlineservice/integration_tests/TicketControllerIntegrationTest.java

```diff
@@ -0,0 +1,100 @@
+package com.awesomeorg.airlineservice.integration_tests;
+
+
+import com.awesomeorg.airlineservice.AbstractIntegrationTest;
+import com.awesomeorg.airlineservice.entity.Ticket;
+import com.awesomeorg.airlineservice.protocol.TicketQuery;
+import com.awesomeorg.airlineservice.protocol.UpdateTicketRequest;
+import com.awesomeorg.airlineservice.service.TicketService;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.time.LocalDate;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+@AutoConfigureMockMvc
+public class TicketControllerIntegrationTest extends AbstractIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private TicketService ticketService;
+
+    private final Ticket existingTicket = new Ticket();
+
+    @Test
+    @SneakyThrows
+    @DirtiesContext
+    public void testCreateTicket() {
+        Ticket ticketToCreate = new Ticket();
+        ticketToCreate.setId(1L);
+        ticketToCreate.setDateOfFlight(LocalDate.now().plusDays(5));
+        ticketToCreate.setDateOfPurchase(LocalDate.now());
+        ticketToCreate.setDateOfReturn(LocalDate.now().plusDays(10));
+        ticketToCreate.setPriceOfTicket((int) 1500.60);
+        ticketToCreate.setSeat(5);
+
+        given(ticketService.createTicket(any(TicketQuery.class))).willReturn(ticketToCreate);
+
+        mockMvc.perform(MockMvcRequestBuilders.post("/tickets")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("{\"dateOfFlight\":\"" + ticketToCreate.getDateOfFlight() + "\"," +
+                                "\"dateOfReturn\":\"" + ticketToCreate.getDateOfReturn() + "\"," +
+                                "\"seat\":" + ticketToCreate.getSeat() + "," +
+                                "\"priceOfTicket\":" + ticketToCreate.getPriceOfTicket() + "}"))
```

Comment:
Also why this? you have mapper, use it to convert java object into the json string